src/database/migrations/1736838705897-create_shortened_urls_table.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateShortenedUrlsTable1736838705897 implements MigrationInterface {
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		await queryRunner.query(`
+            CREATE TABLE "shortened_urls" (
+            id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+            user_id uuid NOT NULL,
+            original_url character varying NOT NULL,
+            short_code character varying(8) NOT NULL,
+            expires_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW() + INTERVAL '1 day',
+            created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+            updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+            deleted_at TIMESTAMP WITH TIME ZONE,
+            CONSTRAINT FK_shortened_urls_user FOREIGN KEY (user_id)
+                REFERENCES users (id)
+                ON DELETE CASCADE
+                ON UPDATE CASCADE,
+            CONSTRAINT unique_shortened_urls_short_code UNIQUE (short_code) 
```

Comment:
```suggestion
            CONSTRAINT UQ_shortened_urls_short_code UNIQUE (short_code) 
```

The convention is to prefix unique constraints by UQ (or UC which is an older convention)

Ref: https://www.scholarhat.com/tutorial/sqlserver/sql-server-naming-conventions-and-standards#:~:text=The%20naming%20conventions%20for%20unique,ColumnName(s)%3E%22.

Reply:
but in the article you provided previously suggested to implement the current convention. Here is that article you provided: https://forge.etsi.org/rep/help/development/database/constraint_naming_convention.md

Reply:
I see... since this is already implemented it should work for now.

In the future it will be a good convention to use `UQ` instead

Reply:
sure, i will create a ticket for it before final deployment